### DISCUSSION TITLE
Math formulas in budibase DB

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7628] [Help] [answered]
I want to calculate amount after applying some percentage of discount 
I want to create some mathematical equations like (a*b)/100 or any other type like a+b-c/d how can I create this by using Budibase DB formula field

**Comment by JayP718** [comment, marked as answer]
Hi,

This is a good starting point for doing basic calculations within Budibase. 

https://docs.budibase.com/docs/formula

Looks like most of what you want to do can be done using handlebars. 

https://docs.budibase.com/docs/bindings

This is a good guide which describes how they work.

Hope this helps!
